tools/qdstat.in

```diff
@@ -540,8 +540,12 @@ class BusManager(object):
                 row.append(PlainNum(node.routerLink))
 
             if self.opts.verbose:
-                row.append(get(node, 'protocolVersion'))
-                row.append(get(node, 'cost'))
+                row.append(PlainNum(get(node, 'protocolVersion')))
+                cost = get(node, 'cost')
+                if cost:
```

Comment:
Why the need for the conditional?  what is the expected output of the 'else' if cost is '0', or None?

Reply:
If the cost is zero or None, the output shows blank string

Reply:
But that isn't clear from the code on line 548:  to me I read

548:  row.append(cost)

And I think it's going to output either "0" or "None".

Change line 548 to:

row.append("")

to make it obvious that the output is to be blank.